src/components/Accordion/Accordion.stories.tsx

```diff
@@ -129,7 +129,7 @@ export const AccordionSubmenu = (props: AccordionProps) => {
         id="languageselector"
         label={<NavigationItem label={currentLanguage}></NavigationItem>}
       >
-        <>
+        <div style={{ paddingLeft: 'var(--spacing-sm)' }}>
```

Comment:
could add tailwind for stuff like this 😄

Reply:
🤮